### PR TITLE
Fix submit review form

### PR DESCRIPTION
**Pull request by zeripath**
Restore the large width of the submit review form, and set a smaller minimum height on mobile. 

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

File: web_src/less/_base.less [hunk 1]
```diff
@@ -1232,9 +1232,19 @@ i.icon.centerlock {
...
+    max-width: 340px;
     overflow-x: hidden;
     text-overflow: ellipsis;
+
+    .editor-toolbar {
+        overflow-x: overlay;
+    }
+
+    @media only screen and (max-width: 768px) {
+        .CodeMirror,.CodeMirror-scroll{
```
Comment (jolheiser) [inline web_src/less/_base.less]:
```suggestion
        .CodeMirror,
        .CodeMirror-scroll {
```
I think this is what the linter wants.

File: web_src/less/_base.less [hunk 2]
```diff
@@ -1231,12 +1231,21 @@ i.icon.centerlock {
 
 /* limit width of all direct dropdown menu children */
 /* https://github.com/go-gitea/gitea/pull/10835 */
-.dropdown:not(.selection) > .menu > * {
-    max-width: 300px;
+.dropdown:not(.selection) > .menu:not(.review-box) > * {
+    max-width: 340px;
```
Comment (zeripath) [inline web_src/less/_base.less]:
```suggestion
    max-width: 300px;
```

**Review by 6543 — APPROVED** [review]
tested, works :tada:

**Review by CirnoT — APPROVED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Comment by silverwind** [comment]
Limiting dropdowns globally probably was a bad idea.

**Pull request merged**